cli/azd/internal/runcontext/cloudshell.go

```diff
@@ -6,10 +6,14 @@ import (
 	"strconv"
 )
 
-const cUseCloudShellAuthEnvVar = "AZD_IN_CLOUDSHELL"
+// AzdInCloudShellEnvVar is the environment variable that is set when running in Cloud Shell. It is set to a value recognized
+// by strconv.ParseBool.
+//
+// Use [IsRunningInCloudShell] to check if the current process is running in Cloud Shell.
+const AzdInCloudShellEnvVar = "AZD_IN_CLOUDSHELL"
 
 func IsRunningInCloudShell() bool {
-	if azdInCloudShell, has := os.LookupEnv(cUseCloudShellAuthEnvVar); has {
+	if azdInCloudShell, has := os.LookupEnv("AZD_IN_CLOUDSHELL"); has {
```

Comment:
Do we prefer using `AzdInCloudShellEnvVar` here?

Reply:
Yes - That's much better!